platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/IDomPopupFrame.java

```diff
@@ -0,0 +1,8 @@
+package com.softicar.platform.dom.elements.popup;
+
+import com.softicar.platform.dom.parent.IDomParentElement;
+
+public interface IDomPopupFrame extends IDomParentElement {
```

Comment:
I introduced it for the sake of symmetry: Otherwise, `DomPopupFrame` would have been the only non-primitive, non-enum, non-interface parameter of any method in `IDomEngine`.

Reply:
then we should probably use it in more places